02-peer-protocol.md

```diff
@@ -79,7 +83,7 @@ desire to set up a new channel.
 
 1. type: 32 (`open_channel`)
 2. data:
-   * [8:temporary-channel-id]
+   * [32:temporary-channel-id]
```

Comment:
How is this to be created? As it's large enough I guess nodes could just use some `chacha20` based CSPRNG...